ccvm_simulators/ccvmplotlib/problem_metadata/boxqp_metadata.py

```diff
@@ -159,32 +114,73 @@ def generate_plot_data(self) -> pd.DataFrame:
                 matching_df = self.__df.loc[self.__df["problem_size"] == problem_size]
                 for percentile in self.__percentile_list[:-1]:
                     sampler = SampleTTSMetric(
-                        tau_attribute="solve_time",
+                        tau_attribute="time",
                         percentile=int(percentile),
                         seed=1,
-                        num_bootstraps=1000,
-                    )
-                    (
-                        best_known_energy_arr,
-                        result_list,
-                        success_prob_arr,
-                    ) = self.generate_arrays_of_TTS(
-                        matching_df, percent_gap, problem_size
+                        num_bootstraps=100,
                     )
 
-                    success_prob_mean = np.mean(
-                        np.array(
-                            [float(success_prob) for success_prob in success_prob_arr]
+                    if TTSType(TTS_type) == TTSType.wallclock:
+                        machine_time = np.mean(matching_df["solve_time"].values)
+                    elif TTSType(TTS_type) == TTSType.physical:
+                        pp_time = np.mean(matching_df["pp_time"].values)
+                        iterations = matching_df["iterations"].values
+                        roundtrip_time = (
+                            (
+                                device_parameters["FPGA_fixed"]
+                                + device_parameters["FPGA_var_fac"]
+                                * float(problem_size)
+                            )
+                            * device_parameters["FPGA_clock"]
+                            + float(problem_size) * device_parameters["laser_clock"]
+                            + device_parameters["buffer_time"]
```

Comment:
This calculation of `roundtrip_time` is specific to the MF-CCVM solver. For the DL-CCVM solver, it should be written as:
```
roundtrip_time = float(problem_size) * device_parameters["laser_clock"]
```

Reply:
Does this mean that when we generate TTS data for plotting, the plotter needs to identify which solver was used because the `rounttrip_time` calculation depends on it?

Reply:
This `machine_time` calculation is replaced with a callback function. Thus, the `roundtrip_time` will be moved into the callback function.